pkg/kubelet/kubelet.go

```diff
@@ -1607,7 +1607,7 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	pullSecrets := kl.getPullSecretsForPod(pod)
 
 	// Call the container runtime's SyncPod callback
-	result := kl.containerRuntime.SyncPod(pod, apiPodStatus, podStatus, pullSecrets, kl.backOff)
+	result := kl.containerRuntime.SyncPod(pod, existingStatus, podStatus, pullSecrets, kl.backOff)
```

Comment:
add a comment why you use existingStatus in the code here instead of apiPodStatus.  i find existing status versus apiPodStatus confusing as well.  one should be previousStatus the other should be observedStatus, can we clean that up?